ADVANCED.md

```diff
@@ -306,54 +305,136 @@ import { SearchBox } from "@elastic/react-search-ui";
 
 ### Example using Autocompleted Suggestions
 
+"Suggestions" are different than "Results". Suggestions are suggested queries. The
+default behavior for suggestions when selected is to perform the search in
```

Comment:
> "Suggestions" are different than "Results". Suggestions are suggested queries. The default behavior for suggestions when selected is to perform the search in the main search page, refreshing the search results.

☠️ Arrr this be a chewy sentence.

Consider a breakdown for clarity:

> "Suggestions" are different than "Results". Suggestions are suggested queries. Unlike an Autocomplete Result, a Suggestion does not go straight to a result page when selected. It acts as a regular search query and refreshes the result set.

Reply:
Used your suggestion.